test/benchmark/load_worlds.py

```diff
@@ -0,0 +1,24 @@
+def run_load_worlds_benchmark():
+    """List worlds and their load time.
+    Note that any first-time imports will be attributed to that world, as it is cached afterwards.
+    Likely best used with isolated worlds to measure their time alone."""
+    import logging
+
+    from Utils import init_logging
+
+    # get some general imports cached, to prevent it from being attributed to one world.
+    import BaseClasses, Launcher, Fill
+
+    from worlds import world_sources
+
+    init_logging("Benchmark Runner")
+    logger = logging.getLogger("Benchmark")
+
+    for module in world_sources:
+        logger.info(f"{module} took {module.time_taken:.4f} seconds.")
+
+
+if __name__ == "__main__":
+    from path_change import change_home
+    change_home()
+    run_load_worlds_benchmark()
```

Comment:
This seems odd in combination with `__init__.py`, which already runs tests on import currently (unless changed to split into init() and run())

Reply:
It doesn't run on import though; as it has a name is main guard?

Reply:
oh, i misread the diff, nvmd this then.